package.json

```diff
@@ -9,11 +9,25 @@
     "url": "https://github.com/Anonyfox/node-webkit-hipster-seed"
   },
   "scripts": {
-    "serve": "node_modules/.bin/bower install && brunch watch --server",
+    "prestart": "node_modules/.bin/bower install",
```

Comment:
Most of these task names are quite confusing. Those removed scripts (sh && bat) were more self-explanatory.